projects/packages/my-jetpack/_inc/state/reducers.js

```diff
@@ -92,6 +94,25 @@ const purchases = ( state = {}, action ) => {
 	}
 };
 
+const chatAvailability = ( state = {}, action ) => {
+	switch ( action.type ) {
+		case SET_CHAT_AVAILABILITY_IS_FETCHING:
+			return {
+				...state,
+				isFetching: action.isFetching,
+			};
+
+		case SET_CHAT_AVAILABILITY:
+			return {
+				...state,
+				items: action?.chatAvailability || [],
```

Comment:
I think another property name (instead of `items`) would reflect better the purpose of this. Also, `|| []` could be `|| {}` as well.

Reply:
Thinking about it again, it could actually be simplified to a bool directly:

```suggestion
				isAvailable: action?.chatAvailability?.is_available || false,
```